projects/backend/capi/atoms.ts

```diff
@@ -35,3 +61,49 @@ export const renderAtom = async (
         js: result['js'],
     }
 }
+
+export const renderAtomElement = async (
+    data: IContentAtomElementFields | undefined,
+    atoms: { [key: string]: IAtom[] },
+): Promise<BlockElement> => {
+    if (data == null) throw new Error('No atom data in element.')
+
+    const atomType = data.atomType
+    const atomId = data.atomId
+    const atom = (atoms[atomType] || []).find(atom => atom.id === atomId)
+    if (atom === undefined) throw new Error('Atom not found in CAPI response.')
+    if (atomType === 'media' && atom !== null) {
+        const imageURL = oc(atom).data.media.posterImage.master.file()
```

Comment:
Can we have a better name for `oc`?

Reply:
it's the name of the library, so renaming might be counterproductive